src/Server.cc

```diff
@@ -182,6 +182,12 @@ Server::Server(const ServerConfig &_config)
     return;
   }
 
+  if (this->dataPtr->sdfRoot.WorldCount() == 0)
+  {
+    ignerr << "SDF file doesn't contain a world.\n";
```

Comment:
It would be helpful to add that if the user wants to spawn a model, they could use the ResourceSpawner GUI plugin or the `world/<world name/create` service.

Reply:
Done in bee3130.